source/direct_debit/index.html.md.erb

```diff
@@ -292,11 +299,23 @@ This would return payments 501 to 1000.
     </tr>
     <tr>
     <td nowrap><code>success</code></td>
-    <td>The payment succeeded.</td>
+    <td>GoCardless has taken the payment from your user’s bank account.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>paidout</code></td>
+    <td>GoCardless has sent the money to your bank account.</td>
+    </tr>    
+    <tr>
+    <td nowrap><code>cancelled</code></td>
+    <td>The payment was cancelled, for example because your user asked their bank to cancel the mandate.</td>
     </tr>
     <tr>
     <td nowrap><code>failed</code></td>
-    <td>The payment failed.</td>
+    <td>The payment failed, for example because your user did not have enough money in their bank account.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>indemnityclaim</code></td>
+    <td>After the payment succeeded, your user asked their bank for a refund ('indemnity claim') under the Direct Debit guarantee. <a href="/support_contact_and_more_information/#contact-us">Contact us</a> if you need to handle this type of refund.</td>
```

Comment:
Because it’s a Very Important Thing, it’s always written as the “​​​Direct Debit Guarantee” with a capital “G”.

Reply:
I've checked and our style seems to be lowercase 'g' - see https://www.payments.service.gov.uk/direct-debit/ for example.

Reply:
I think we’re inconsistent: our payment pages and the emails we send to users use a capital “G”. The Direct Debit scheme itself always seems to use a capital “G” (see https://www.directdebit.co.uk/DirectDebitExplained/pages/directdebitguarantee.aspx for example).

Reply:
Makes sense - will tweak, thanks Alex!

Reply:
Fixed in https://github.com/alphagov/pay-tech-docs/pull/306/commits/f1c596cb72e27227e852ea0ab29894f612865de8.